README.md

```diff
@@ -20,3 +20,7 @@ There is currently no official installer for Earth Enterprise Fusion & Server.
 
 ### Known Issues
 Known issues can be tracked via [wiki page](https://github.com/google/earthenterprise/wiki/Known-Issues), which is updated weekly.
+
```

Comment:
The PR for adding the gitter badge is adding it at the top of the README. I'm not sure which one I like better, but they should be consistent.